sections/semantics-document-metadata.include

```diff
@@ -826,18 +826,17 @@
   <p><a href="https://www.w3.org/TR/css-device-adapt-1/#viewport-meta"><code>&lt;meta name="viewport" content="..."></code></a>
   allows authors to define specific viewport characteristics (such as the layout viewport's width and zoom factor)
   for their documents. Among these is the ability to prevent or restrict users from being able to zoom, using
-  <code>content</code> values such as <code>user-scalable=no</code> or <code>maximum-scale=1.0</code>. Authors should not suppress
+  <code>content</code> values such as <code>user-scalable="no"</code> or <code>maximum-scale=1.0</code>. Authors should not suppress
   or limit the ability of users to resize a document, as this causes accessibility and usability issues.</p>
 
   <div class="example">
   The following examples illustrate code that should be avoided:
   <pre highlight="html">
```

Comment:
should this use class="bad"?

Reply:
Yes - but `class="bad"` doesn't seem to do anything when run through bikeshed. That is, there's no visual difference.